code/tests/shopping.R

```diff
@@ -6,7 +6,7 @@ devtools::install_github("robinlovelace/simodels")
 library(simodels)
 source("R/gravity_model.R")
 
-disag_threshold = 50 # would increasing this reduce the number of od pairs?
+disag_threshold = 1000 # increasing this reduces the number of od pairs
```

Comment:
Correct but 1000 is really high, meaning that up to 1000 trips could take place between an origin and destination without any disaggregation. I suspect there are other ways to reduce the number of desire lines, have you tried reducing the number of destinations?